k8s/extensions.deployment.yaml

```diff
@@ -0,0 +1,73 @@
+kind: Namespace
```

Comment:
Would you rebase? I added k8s/extensions.controller.yaml and renamed k8s/deployment.yaml to k8s/controller.yaml in HEAD. You can remove both changes in k8s/.